sign/bls/bls_test.go

```diff
@@ -0,0 +1,18 @@
+package bls
+
+import (
+	"testing"
+
+	"github.com/dedis/kyber/pairing/bn256"
+	"github.com/dedis/kyber/util/random"
+)
+
+func TestBLS(t *testing.T) {
+	msg := []byte("Hello Boneh-Lynn-Shacham")
+	suite := bn256.NewSuite()
+	private, public := NewKeyPair(suite, random.New())
+	sig := Sign(suite, private, msg)
+	if err := Verify(suite, public, msg, sig); err != nil {
```

Comment:
Could you also write a test that changes the signature to random bytes and "verifies that it does not verify" ? ;)